conjur/api/client.py

```diff
@@ -180,6 +181,12 @@ def get_many(self, *variable_ids) -> Optional[bytes]:
         """
         return self._api.get_variables(*variable_ids)
 
+    def create_token(self, create_token_data: CreateTokenData) -> str:
+        """
+        Create token/s for hosts with restrictions
```

Comment:
we create only a single token, correct?
```suggestion
        Create token for hosts with restrictions
```

Reply:
You pass an argument count